src/envs/cartpole.jl

```diff
@@ -1,9 +1,24 @@
-
+module CartPoleEnv
 # Ported from: https://github.com/openai/gym/blob/996e5115621bf57b34b9b79941e629a36a709ea1/gym/envs/classic_control/cartpole.py
 # which has header:
 # 		Classic cart-pole system implemented by Rich Sutton et al.
 #		Copied from https://webdocs.cs.ualberta.ca/~sutton/book/code/pole.c
 
+using Reinforce: AbstractEnvironment
+using LearnBase: DiscreteSet
+using RecipesBase
+using Random: seed!
+
+import Reinforce: reset!, actions, finished, step!
+
+export
+  CartPole,
+  reset!,
+  step!,
+  actions,
+  finished,
+  f
```

Comment:
`f` ?
I think recipe can work without exporting it. Could you test that?

Reply:
Yup, this works. Committed.